homeassistant/components/smartenergy_awattar/config_flow.py

```diff
@@ -0,0 +1,63 @@
+"""Awattar config flow and options flow setup."""
+
+from typing import Any, Literal
+
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.const import CONF_SCAN_INTERVAL
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import CONF_COUNTRY, CONF_COUNTRY_LIST, DOMAIN
+
+
+def _get_config_schema(default_values: dict) -> vol.Schema:
+    """Define a schema with default values and return it."""
+    return vol.Schema(
+        {
+            vol.Required(
+                CONF_COUNTRY,
+                default=default_values.get(CONF_COUNTRY, CONF_COUNTRY_LIST[0]),
+            ): vol.In(sorted(CONF_COUNTRY_LIST)),
+            vol.Optional(
+                CONF_SCAN_INTERVAL,
+                default=default_values.get(CONF_SCAN_INTERVAL, 10),
+            ): vol.All(vol.Coerce(int), vol.Range(10, 60000)),
```

Comment:
HA is supporting [selectors](https://www.home-assistant.io/docs/blueprint/selectors).
In this case, I would suggest a [number selector](https://www.home-assistant.io/docs/blueprint/selectors/#number-selector).

For implementation see also https://developers.home-assistant.io/docs/data_entry_flow_index#show-form